nmosnode/aggregator.py

```diff
@@ -242,9 +256,9 @@ def _process_reregister(self):
         except InvalidRequest as e:
```

Comment:
Can this (and other calls to _SEND) not result in an InvalidTokenError which needs handling?

Reply:
Yeah the exception handling should be moved into the _SEND function ideally, good spot